.github/actions/bouf/action.yaml

```diff
@@ -0,0 +1,139 @@
+name: Run bouf
+description: Generates signed OBS install files and updater files
+inputs:
+  gcpWorkloadIdentityProvider:
+    description: GCP Identity Provider Pool ID
+    required: true
+  gcpServiceAccountName:
+    description: Google service account name
+    required: true
+  version:
+    description: Version string (e.g., 30.0.0-rc1)
+    required: true
+  channel:
+    description: Update channel
+    required: false
+    default: 'stable'
+
+runs:
+  using: composite
+  steps:
+    - name: Extract Artifact
+      shell: pwsh
+      run: |
+        Expand-Archive -Path build\*.zip -DestinationPath build
+        Remove-Item build\*.zip
+
+    - name: Setup bouf
+      shell: pwsh
+      env:
+        BOUF_TAG: 'v0.6.0-nsis-update'
+        BOUF_HASH: '892640b3322046d28f0d2ef9dd24d3ed45499dca39852c6ce70cba44269ca2d0'
+        BOUF_NSIS_HASH: '790edd61161b76b74c71fe68892bcbf4357e0d83ee67f3bbfd95af0e4d609515'
+        GH_TOKEN: ${{ github.token }}
+      run: |
+        # Download bouf release
+        . $env:GITHUB_ACTION_PATH\Ensure-Location.ps1
+        . $env:GITHUB_ACTION_PATH\Invoke-External.ps1
+        Ensure-Location bouf
+        $windows_zip = "bouf-windows-$env:BOUF_TAG.zip"
```

Comment:
We usually enclose variables for interpolation with braces (similar to Bash) to ensure the variable name is clearly scoped (e.g. `${env:BOUF_TAG}` or `$($Object.Method())`).

Outside of interpolation there is usually no need for enclosing (or even quoting in some cases).